css/index.css

```diff
@@ -138,9 +206,60 @@ footer nav {
     align-items: center;
     padding: 20px 2%;
     font-size: 14px;
-}
 
-footer nav a {
-    color: white;
+}
+footer nav a{
     text-decoration: none;
-}
\ No newline at end of file
+    color: white;
+}
+
+.boxes .box:first-child{
+    background: teal;
+
+}
+
+.boxes .box:nth-child(2){
+ background: gold;
+}
+.boxes .box:nth-child(3){
```

Comment:
great job using CSS Selectors and having the styles cascade rather than re-writing classes